.github/workflows/codeql-analysis.yml

```diff
@@ -35,7 +35,7 @@ jobs:
         uses: ./.github/actions/environment
 
       - name: Initialize CodeQL
-        uses: github/codeql-action/init@1b549b9259bda1cb5ddde3b41741a82a2d15a841 # pin@v2
+        uses: github/codeql-action/init@45775bd8235c68ba998cffa5171334d58593da47 # pin@v2
```

Comment:
question: Which actions do we pin?

And which actions do we use floating versions with?
https://github.com/getsentry/sentry-dotnet/blob/f08acb30d1e2f22c35be7bfba626c1b406e1345a/.github/workflows/codeql-analysis.yml#L30

Reply:
Generally we try to pin everything. If we want to change the version of an action it should be via a change/pr not magically behind the scenes without us even realising. Then we have these bots that tell us when new versions of things are available and put together a PR specifically for that bump (to make sure everything is green before merging into the main branch). This keeps us up to date whilst maintaining stability in the main branch.